testsuite/setup.py

```diff
@@ -123,6 +123,8 @@
                   'data/dlpoly/CONFIG*',
                   'data/dlpoly/HISTORY*',
                   'data/*.xml',
+                  'coordinates/data/*xyz',
```

Comment:
Well I thought that this file was not usable for anything else. Here we might also purposely add miss-formatted or broken files to check if the correct exceptions are raised.  I know we used to try to use the same files all over. Personally I prefer special files with values that I can calculate analytically before.